release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2682,3 +2682,28 @@ link:https://access.redhat.com/solutions/6828761[{product-title} 4.9.25 containe
 ==== Updating
 
 To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-26"]
+=== RHBA-2022:1022 - {product-title} 4.9.26 bug fix and security update
+
+Issued: 2022-03-29
+
+{product-title} release 4.9.26 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:1022[RHBA-2022:1022] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:1021[RHSA-2022:1021] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6843421[{product-title} 4.9.26 container image list]
+
+[id="ocp-4-9-26-bug-fixes"]
+==== Bug fixes
+
+* Previously, TCP connections would not close after completing health checks in the Ingress Operator. Consequently, TCP connections would accumulate, causing a buildup in the `LoadBalancer`. This fix disables `keepalive` while establishing a connection. This results in the TCP connection closing after each health check, preventing buildup in the `LoadBalancer`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2064586[*BZ#2064586*])
+
+* Previously, low limit numbers for client throttling messages would appear while running clusters. Consequently, the increased number of custom resource definitions (CRDs) generated low numbers and restricted API discovery requests. This fix increases the limit number, resulting in messages appearing less frequently. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2045008[*BZ#2045008*])
+
+* Previously, the `oc debug` command did not allow users to set a connection timeout. Consequently, users were never logged out of the cluster environment. This fix adds a timeout feature to shut down a cluster after a period of inactivity. This results in clusters automatically closing after a prolonged period of idleness to offset the `oc debug` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065302[*BZ#2065302*])
```

Comment:
```suggestion
* Previously, the `oc debug` command did not allow users to set a connection timeout, and users were never logged out of the cluster environment. With this update,  a timeout feature shuts down a cluster after a period of inactivity. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2065302[*BZ#2065302*])
```
Just a suggestion for this one. I removed the last sentence bc it says pretty much the same thing as the sentence before. Take it or leave it. :)